openedx/core/lib/xblock_utils.py

```diff
@@ -240,7 +240,7 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
         else:
             return frag
 
-    if isinstance(block, SequenceModule):
+    if isinstance(block, SequenceModule) or getattr(block, 'HIDDEN', False):
```

Comment:
Actually, I'd lean towards leaving the staff debug info available to instructors, so that they can see that

1. the block is hidden
2. interact with student state for the block

@explorerleslie, what do you think?